consensus/src/chained_bft/safety/vote_msg.rs

```diff
@@ -105,6 +113,10 @@ impl VoteMsg {
         proposed_block_id: HashValue,
         executed_state: ExecutedState,
         round: Round,
+        parent_block_id: HashValue,
```

Comment:
we probably should unify the name between vote_msg and quorum_cert, it's parent_ and grandparent_ here but two_chain/three_chain in qc

Reply:
Unified the naming to parent and grandparent